src/main/java/no/nav/pto/veilarbportefolje/fargekategori/FargekategoriController.java

```diff
@@ -37,12 +37,17 @@ public class FargekategoriController {
     @PostMapping("/hent-fargekategori")
     public ResponseEntity<FargekategoriEntity> hentFargekategoriForBruker(@RequestBody HentFargekategoriRequest request) {
         validerRequest(request.fnr);
-        NavKontor navKontorForBruker = brukerServiceV2.hentNavKontor(request.fnr).orElseThrow(() -> new ResponseStatusException(HttpStatus.FORBIDDEN, "Bruker er ikke tilordnet enhet"));
+        NavKontor navKontorForBruker = brukerServiceV2.hentNavKontor(request.fnr).orElseThrow(() -> new ResponseStatusException(HttpStatus.FORBIDDEN, "Bruker har ikke tilordnet enhet"));
 
         authService.innloggetVeilederHarTilgangTilOppfolging();
         authService.innloggetVeilederHarTilgangTilBruker(request.fnr.get());
         authService.innloggetVeilederHarTilgangTilEnhet(navKontorForBruker.getValue());
+
         try {
+            if (!harBrukerenTildeltVeileder(request.fnr)) {
```

Comment:
we have to return it for the advarsel om sletting in frontend.

Reply:
Hm, but maybe we need to create another endpoint for that check..